src/platform/EFR32/freertos_bluetooth.c

```diff
@@ -71,27 +78,36 @@ sl_status_t bluetooth_start(UBaseType_t ll_priority, UBaseType_t stack_priority,
     bluetooth_event_flags = xEventGroupCreate();
     configASSERT(bluetooth_event_flags);
 
-    BluetoothMutex = xSemaphoreCreateMutex();
+    BluetoothMutex = xSemaphoreCreateMutexStatic(&bluetoothMutexStruct);
 
     err = initialize_bluetooth_stack();
 
     if (err == SL_STATUS_OK)
     {
         // create tasks for Bluetooth host stack
-        xTaskCreate(BluetoothTask,                              /* Function that implements the task. */
-                    "Bluetooth Task",                           /* Text name for the task. */
-                    BLUETOOTH_STACK_SIZE / sizeof(StackType_t), /* Number of indexes in the xStack array. */
-                    NULL,                                       /* Parameter passed into the task. */
-                    stack_priority,                             /* Priority at which the task is created. */
-                    &BluetoothTaskHandle);                      /* Variable to hold the task's data structure. */
+        BluetoothTaskHandle =
+            xTaskCreateStatic(BluetoothTask,                              /* Function that implements the task. */
+                              "Bluetooth Task",                           /* Text name for the task. */
+                              BLUETOOTH_STACK_SIZE / sizeof(StackType_t), /* Number of indexes in the xStack array. */
```

Comment:
```suggestion
                              ArraySize(bluetoothStack),                  /* Number of indexes in the xStack array. */
```

Reply:
Seems like CodeUtils.h is not c file friendly so I'll keep this as is.

Reply:
Fun.  I filed https://github.com/project-chip/connectedhomeip/issues/5667 to fix this so we can use it here.